superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -219,6 +219,7 @@ const SqlEditor = ({
     if (latestQuery && ['running', 'pending'].indexOf(latestQuery.state) >= 0) {
       dispatch(postStopQuery(latestQuery));
     }
+    return false;
```

Comment:
What is this used for?

Reply:
yes to prevent the event from bubbling up